pkg/rpc/client/nep11.go

```diff
@@ -83,8 +87,22 @@ func (c *Client) CreateNEP11TransferTx(acc *wallet.Account, tokenHash util.Uint1
 }
 
 // NEP11TokensOf returns an array of token IDs for the specified owner of the specified NFT token.
+// It uses iterator-based sessions to retrieve iterator values.
 func (c *Client) NEP11TokensOf(tokenHash util.Uint160, owner util.Uint160) ([][]byte, error) {
-	result, err := c.InvokeAndPackIteratorResults(tokenHash, "tokensOf", []smartcontract.Parameter{
+	return c.nep11TokensOf(tokenHash, owner, c.InvokeFunction)
```

Comment:
It's not correct (and in other places too), this method can no longer return `[][]byte`, instead it should return an iterator for clients to use with traversing methods.

Reply:
Fixed.